portfolio/src/main/webapp/style.css

```diff
@@ -16,4 +16,9 @@ body::before {
 /* Turn secret heart button red on hover */
 .bi-heart-fill:hover{
   color: red !important;
+}
+
+#forum-area{
```

Comment:
[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space between the last selector and the declaration block.